recipe/meta.yaml

```diff
@@ -12,25 +12,22 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  skip: false
-  number: 0
+  number: 1
 
 requirements:
   build:
     - python
     - setuptools
     - toolchain
     - future
-    - numpy x.x
+    - numpy 1.11
```

Comment:
Looks like this should be 1.11.* as per [docs](https://conda-forge.org/docs/meta.html#building-against-numpy)